api/src/main/java/com/netflix/iceberg/expressions/Predicate.java

```diff
@@ -62,6 +62,8 @@ public String toString() {
         return String.valueOf(ref()) + " == " + literal();
       case NOT_EQ:
         return String.valueOf(ref()) + " != " + literal();
+      case STARTS_WITH:
+        return "startsWith(" + ref() + "," + literal() + ")";
```

Comment:
I wonder if it would be more clear to use `ref() + "LIKE '" + literal() + "%'"` instead of this. Or maybe just use `startsWith` as an infix operator to avoid adding the SQL wildcard character, `%`.

Reply:
I agree with not adding a reference to SQL. 
I like the option of using `startsWith` as an infix operator, but then we should update the `IS_NULL` and the `NOT_NULL` to not to use parenthesis? IMHO I would stick denoting them as "functions" (using parenthesis).

Reply:
Let's keep those as functions. The difference here is that startsWith(a, b) doesn't tell us whether this means "a starts with b" or "b starts with a". Using infix notation is just easier to understand.

Reply:
Thanks, I updated accordingly.